### ISSUE TITLE
GTF specification should include start and stop codons

### ISSUE THREAD

**Issue by ewallace** [xryanglab/RiboCode#21]
I think, the input `gtf` to `metaplots` and `prepare_transcripts` needs `start_codon` and `stop_codon` features in order to recognize something as an annotated CDS; `CDS` features are not enough. Could this be added as an option in `GTF_update.rst`?

Importantly, could this be clarified explicitly in the gtf specification in `README.md`?

**Comment by zhengtaoxiao** [comment]
Yes, the genomic coordinates of start_codon and stop_codon are required for metaplots. "GTF_update" will use these two features if they are provided in the GTF file, otherwise the "prepare_transcripts" will try to infer them from the "CDS" feature. We will explain it more clearly in the next version. Thanks!
